### PR TITLE
MISUV-5856 SCP025 | Add UK property | Build - obligations (UK property added)

### PR DESCRIPTION
**Pull request by EdwardDennis**

### CODE REVIEW

File: it/connectors/NavBarEnumFsConnectorISpec.scala
Comment (tharunprasath) [inline it/connectors/NavBarEnumFsConnectorISpec.scala:1]: Test to be removed?

File: test/controllers/incomeSources/add/UKPropertyAddedControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      }
+    }
+    "return 303 SEE_OTHER" when {
+      "Income Sources FS is disabled" in {
+        disable(IncomeSources)
+        setupMockAuthRetrievalSuccess(BaseTestConstants.testIndividualAuthSuccessWithSaUtrResponse())
+        mockUKPropertyIncomeSource()
+
+        val result = TestUKPropertyAddedController.show(testPropertyIncomeId)(fakeRequestWithActiveSession)
+        status(result) shouldBe SEE_OTHER
```
Comment (tharunprasath) [inline test/controllers/incomeSources/add/UKPropertyAddedControllerSpec.scala:108]: Is redirecting to the homepage expected?

**Review by tharunprasath — CHANGES_REQUESTED** [review]

File: it/connectors/NavBarEnumFsConnectorISpec.scala
Comment (EdwardDennis) [reply it/connectors/NavBarEnumFsConnectorISpec.scala:1]: Oops

File: conf/app.routes [hunk 2]
```diff
@@ -149,7 +149,12 @@ GET          /agents/how-to-claim-refund                                  contro
 GET          /how-to-claim-refund-external                                controllers.NotMigratedUserController.redirect()
 GET          /agents/how-to-claim-refund-external                         controllers.NotMigratedUserController.redirectAgent()
```
[the remark this answers is not in the document]
Comment (EdwardDennis) [reply conf/app.routes:151]: I think this is fine no? It's more readable this way

File: test/controllers/incomeSources/add/UKPropertyAddedControllerSpec.scala (hunk 1 above)
Comment (EdwardDennis) [reply test/controllers/incomeSources/add/UKPropertyAddedControllerSpec.scala:108]: When the FS is disabled yes

File: conf/app.routes (hunk 2 above)
[the remark this answers is not in the document]
Comment (tharunprasath) [reply conf/app.routes:151]: Ho Sorry, I misunderstood the ADD comment as MERGE auto-added conflict comments.

**Pull request merged**